keyboards/satan/keymaps/olligranlund_ISO/config.h

```diff
@@ -0,0 +1,19 @@
+#include "../../config.h"
```

Comment:
This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary. 

Additionally, could you add `#pragma once` at the top of the file?